server/src/test/java/org/elasticsearch/discovery/ClusterDisruptionIT.java

```diff
@@ -390,10 +387,10 @@ public void testSearchWithRelocationAndSlowClusterStateProcessing() throws Excep
 
     public void testIndexImportedFromDataOnlyNodesIfMasterLostDataFolder() throws Exception {
         // test for https://github.com/elastic/elasticsearch/issues/8823
-        configureCluster(2, 1);
+        configureCluster(2);
         String masterNode = internalCluster().startMasterOnlyNode(Settings.EMPTY);
         internalCluster().startDataOnlyNode(Settings.EMPTY);
-
+        setMinimumMasterNodes(1);
```

Comment:
Is this necessary? I think that the cluster should know it only has one master node and sets this accordingly.

Reply:
You're right, it's not :) Removing.